cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -692,6 +717,18 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
     GradingEditor = AbstractEditor.extend({
         templateName: 'grading-editor',
         className: 'edit-settings-grading',
+        events: {
+            'change #grading_type': 'handleGradingSelect',
+        },
+
+        handleGradingSelect: function(event) {
+            event.preventDefault();
+            if (this.$('#grading_type').val() !== 'notgraded' && course.get('self_paced') && course.get('is_custom_relative_dates_active')) {
+                $('#relative_date_input').show();
+            } else {
+                $('#relative_date_input').hide();
+            }
+        },
```

Comment:
Is it possible to put this in the SelfPacedDueDateEditor?

Reply:
Nope :( Doesn't seem to work if I put it in SelfPacedDueDateEditor

Reply:
All good. Might just be because the SelfPacedDueDateEditor wasn't on the page or something to be able to capture the event. Or could have to do a bunch of other weird internals. All sorted.